.drone.star

```diff
@@ -46,42 +46,47 @@ config = {
             ],
             "webUISharingBasic": [
                 "webUISharingAcceptShares",
-                "webUISharingAcceptSharesToRoot",
```

Comment:
I guess you are deleting this because "sharing to root"  is not a valid thing to do in OCIS.